pkg/scheduler/queue/tenant_queues_test.go

```diff
@@ -836,7 +836,9 @@ func (qb *queueBroker) makeQueuePathForTests(tenantID TenantID) QueuePath {
 }
 
 func isConsistent(qb *queueBroker) error {
-	if len(qb.tenantQuerierAssignments.querierIDsSorted) != len(qb.tenantQuerierAssignments.queriersByID) {
+	if len(qb.tenantQuerierAssignments.querierIDsSorted) != len(qb.querierConnections.queriersByID) {
+		fmt.Println(qb.tenantQuerierAssignments.querierIDsSorted)
+		fmt.Println(qb.querierConnections.queriersByID)
```

Comment:
we probably do not want to leave the prints in, right? If we do we should use a test logger

Reply:
oops